Tests/XMLCoderTests/AdvancedFeatures/AttributedIntrinsicTest.swift

```diff
@@ -104,42 +95,24 @@ private struct FooValue: Codable, Equatable {
     }
 }
 
-private struct FooOptional: Codable, DynamicNodeEncoding, Equatable {
```

Comment:
I understand that this is work in progress, but it would be great to keep `DynamicNodeEncoding` in at least a few more versions, but to mark it as deprecated. Unfortunately, this means that we'll have to test both implementations at the same time. Maybe by keeping two versions of the same type, one declared with `DynamicNodeEncoding`, the other using property wrappers. This is the best way to make things easier for users of XMLCoder so that they have some time to migrate, while still getting other new features and bugfixes with new versions.

Reply:
Totally!